### PR TITLE
PAE- 103: Add the dropdown for orgs but as a custom theme

### PR DESCRIPTION
**Pull request by elvinquero**
This PR has to do with https://github.com/Pearson-Advance/edx-platform/pull/53. That PR alter the default index.html from the studio app, which may be not desirable.

With this PR we added the custom index.html. It checks were the new variable, org_names_list exists to create the dropdown, if not, renders the text input, as the default code does.

To activate this theming we will have to:
1. Enable comprehensive theming for Studio.
2. Enable the ORGANIZATION_APP https://github.com/Pearson-Advance/edx-platform/blob/3792f3d740d76b245f7ff05492ede2ce9a637129/cms/envs/common.py#L283 
3. Add the studio URL to the Site Theme table. 

Antes 
Screenshot from 2021-10-01 16-54-21

Después
Screenshot from 2021-10-06 16-37-05

### CODE REVIEW

File: edx-platform/pearson-pathways-theme/cms/templates/index.html [hunk 1]
```diff
@@ -0,0 +1,597 @@
...
+          <fieldset class="form-group">
+            <div class="field">
+              <label class="field-label">${_("Show all courses in organization:")}
+                % if org_names_list:
+                <select class="form-select" name="org">
+                    <option value="" selected>${_("All organizations")} </option>
+                    %for org in org_names_list:
+                    <option value="${org}">${org}</option>
+                    %endfor
+                  </select>
```
Comment (Jacatove) [inline edx-platform/pearson-pathways-theme/cms/templates/index.html]:
```suggestion
                </select>
```

**Review by Jacatove — CHANGES_REQUESTED** [review]
Great! This way We do not touch the studio app. I have a question, isn't worth it to also add this changes to the other theme within eComm which is **pols**?. @elvinquero

File: edx-platform/pearson-studio-theme/cms/templates/index.html [hunk 2]
```diff
@@ -193,8 +193,17 @@ <h2 class="title title-3">${_("Organization and Library Settings")}</h2>
           <fieldset class="form-group">
             <div class="field">
               <label class="field-label">${_("Show all courses in organization:")}
+                % if org_names_list:
+                <select class="form-select" name="org">
+                    <option value="" selected>${_("All organizations")} </option>
```
Comment (Squirrel18) [inline edx-platform/pearson-studio-theme/cms/templates/index.html]: nit: Remove the blank space before the closing tag. @anfbermudezme

File: edx-platform/pearson-studio-theme/cms/templates/index.html [hunk 3]
```diff
@@ -193,8 +193,17 @@ <h2 class="title title-3">${_("Organization and Library Settings")}</h2>
...
+                    %for org in org_names_list:
+                    <option value="${org}">${org}</option>
+                    %endfor
+                  </select>
```
Comment (Squirrel18) [inline edx-platform/pearson-studio-theme/cms/templates/index.html]: Bad indentation. @anfbermudezme

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

**Comment by elvinquero** [comment]
@Squirrel18 suggestions were applied.

**Review by anfbermudezme — APPROVED** [review]

**Review by diegomillan — APPROVED** [review]
LGTM

**Review by Jacatove — CHANGES_REQUESTED** [review]
@elvinquero Could you please squash the commits. I mean just to have the 2 "feat:..." commits.

**Review by Jacatove — APPROVED** [review]

**Review by Squirrel18 — APPROVED** [review]

**Review by Squirrel18 — APPROVED** [review]

**Pull request merged**